ts/screens/onboarding/OnboardingServicesPreferenceScreen.tsx

```diff
@@ -55,6 +56,9 @@ const OnboardingServicesPreferenceScreen = (
   React.useEffect(() => {
     // when the user made a choice (the profile is right updated), navigate to thank-you page
```

Comment:
```suggestion
    // when the user made a choice (the profile is right updated), continue to the next step
```
This comment is not longer valid